ffs/terms.md

```diff
@@ -0,0 +1,61 @@
+
+In the event that the community does not agree to the new terms, the contractor can make further ammends and repeat the process or choose to not pick up the proposal. He/she may, of course, also choose to do the work as he/she pleases without community input, though they cannot formally charge the FFS. In the event that he/she completes the project on his/her own the community may decide to release funds anyway, but there are no guarentees.
+
+## Investigation
+In the event that a contractor does not comply with the terms above, a special Monero investigative team will be put together that should include one Core team member (or trusted proxy) and at least one other volunteer. The proposal is considered 'on hold' until investigation is complete. Should work continue and milestones be completed during a time of investigation, no payouts are to be made until the investigation is complete.
+
+The contractor is expected to comply with the investigative team's requests, which may include, but are not limited to, chat logs, proof of work, and other requested materials.
+
+The investigative team is write up their conclusion and present to the community, who can then either provide a formal dismissal or allow the contractor to continue the contract.
+
+## Appeal
+In the event that a contract leaves a proposal or is dismissed, but he/she has, in the time that followed, proven their goodwill, they can make an appeal to the community to resume the contract, assuming another contractor hasn't already picked up the proposal. They can provide whatever evidence they think will be helpful in convincing the community to let them resume their previous work.
+
+Should the community decide to allow the contractor to resume his/her work, he/she may begin immediately. Should they decide not to, the contractor is expected not to be a baby about it.
+
+## Conclusion (a note about the community)
+This conclusion is in plain terms and in non-official voice, so is not expected to be taken with the weight of the above terms.
+
+We apologize if these terms seem overly strict or harsh. The community of Monero is really a very welcoming and friendly place. We just want to ensure that we have an environment that is fair, encourages innovation, protects the money of contributors, and protects contractors. In other words, we want a [smooth](https://github.com/iamsmooth) FFS experience for everyone, contractors and contributors alike.
+
+We all want Monero to succeed, and we've got an awesome community that can make it happen. Let's just treat each other well, be honest with each other about anything unexpected that may arise, and make reasonable arguments for or against something, and, you know, be adults. In doing so, we shouldn't have any issues.
+
+Hopefully.
```

Comment:
>We apologize

There's no need to apologize for being professional

>Hopefully.

I don't think the document should end like a personal diary. I think we can get the point across with a disclaimer.